warrior/WarriorCore/Classes/argument_datatype_class.py

```diff
@@ -64,27 +65,32 @@ def convert_arg_to_datatype(self):
             self.datatype = tuple
         elif self.arg_name.startswith('dict_'):
             self.datatype = dict
+        elif self.arg_name.startswith('file_'):
+            self.datatype = file
+            try:
+                self.arg_value = file(self.arg_value)
+            except IOError:
+                pass
 
         else:
-            #print_info("User has not specified any data type,
-            #input argument will be treated as string (default)")
+            print_info("User has not specified any data type, input argument "
+                       "will be treated as string (default)")
```

Comment:
This message will be printed n times if there are n arguments in the step and also this message doesn't convey which input argument is considered here.